ports/ismrmrd/vcpkg.json

```
@@ -1,13 +1,21 @@
 {
   "name": "ismrmrd",
   "version-string": "1.5.0",
```

```suggestion
  "version": "1.5.0",
```
This might be served by `version `here.